src/Form/Form.php

```diff
@@ -116,6 +128,37 @@ public function validator(Validator $validator = null)
         return $this->_validator;
     }
 
+    /**
+     * Get the validator for this form.
+     *
+     * This method will call `_buildValidator()` when the validator
+     * is first built. This hook method lets you configure the
+     * validator or load a pre-defined one.
+     *
+     * @return \Cake\Validation\Validator the validator instance.
+     */
+    public function getValidator()
```

Comment:
Should Form use the `ValidatorAwareTrait`? That would let it implement the same interface as the ORM and open up the option for multiple validators if people need them.

Reply:
@markstory I thought about that but `Form::validator()` and `ValidatorAwareTrait::validator()` don't have the same arguments, so I don't think PHP will be happy if I make `Form` use `ValidatorAwareTrait`.

Reply:
But perhaps at least I can make the new get/set methods have the same params as the trait so that in 4.x we can get rid of the method and have `Form` use `ValidatorAwareTrait`.

Reply:
You could alias the trait method to another name and make it protected?